server/src/main/scala/za/co/absa/atum/server/api/http/BaseEndpoints.scala

```diff
@@ -71,7 +71,8 @@ trait BaseEndpoints {
       oneOf[ErrorResponse](
         badRequestOneOfVariant,
         generalErrorOneOfVariant,
-        internalServerErrorOneOfVariant
+        internalServerErrorOneOfVariant,
+        notFoundErrorOneOfVariant
```

Comment:
Not here in the base endpoint, add this to the GET endpoint for partitioning measures. The base endpoint should provide only things common to every single endpoint.

Reply:
I actually didn't check it. Thank you for the alert, I will remove it.